### DISCUSSION TITLE
Choose a different folder

### DISCUSSION THREAD

**Discussion by qroft** [wintercms/wn-blog-plugin#23] [Ideas]
AFAIK the uploaded images per blog entry are uploaded into a fixed folder.
What do you guys think about having the possibility to select the image from within the media manager?

There is a plugin that offers this for October CMS but the developer will not port it to Winter CMS.

**Comment by LukeTowers** [comment]
You could probably port it yourself if you wanted to. Shouldn't be that difficult. May even work out of the box with Winter.
